cfme/tests/v2v/test_post_migrations.py

```diff
@@ -1,191 +1,100 @@
-    """Test policy to prevent source VM from starting if migration is complete
+@pytest.mark.tier(1)
+def test_migration_post_attribute(appliance, provider, mapping_data_vm_obj_mini, soft_assert):
+    """
+    Test to validate v2v post-migrations usecases
 
     Polarion:
         assignee: ytale
         initialEstimate: 1/4h
+        caseimportance: critical
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
         casecomponent: V2V
+        testSteps:
+            1. Add source and target provider
+            2. Create infra map and migration plan
+            3. Migrate VM
+            4. Check vm attributes upon successful migration
     """
-    infrastructure_mapping_collection = appliance.collections.v2v_mappings
-    mapping = infrastructure_mapping_collection.create(form_data_vm_obj_single_datastore.form_data)
-
-    @request.addfinalizer
-    def _cleanup():
-        infrastructure_mapping_collection.delete(mapping)
-
-    migration_plan_collection = appliance.collections.v2v_plans
-    # vm_obj is a list, with only 1 VM object, hence [0]
-    vm_obj = form_data_vm_obj_single_datastore.vm_list[0]
+    src_vm_obj = mapping_data_vm_obj_mini.vm_list[0]
+    source_view = navigate_to(src_vm_obj, "Details")
+    summary = source_view.entities.summary("Properties").get_text_of("Container")
+    source_cpu, source_socket, source_core, source_memory = re.findall(r"\d+", summary)
+    migration_plan_collection = appliance.collections.v2v_migration_plans
     migration_plan = migration_plan_collection.create(
         name="plan_{}".format(fauxfactory.gen_alphanumeric()),
         description="desc_{}".format(fauxfactory.gen_alphanumeric()),
-        infra_map=mapping.name,
-        vm_list=form_data_vm_obj_single_datastore.vm_list,
-        start_migration=True,
-    )
-
-    # explicit wait for spinner of in-progress status card
-    view = appliance.browser.create_view(
-        navigator.get_class(migration_plan_collection, "All").VIEW.pick()
+        infra_map=mapping_data_vm_obj_mini.infra_mapping_data.get("name"),
+        target_provider=provider,
+        vm_list=mapping_data_vm_obj_mini.vm_list
     )
-    wait_for(
-        func=view.progress_card.is_plan_started,
-        func_args=[migration_plan.name],
-        message="migration plan is starting, be patient please",
-        delay=5,
-        num_sec=150,
-        handle_exception=True,
-        fail_cond=False
-    )
-
-    # wait until plan is in progress
-    wait_for(
-        func=view.plan_in_progress,
-        func_args=[migration_plan.name],
-        message="migration plan is in progress, be patient please",
-        delay=15,
-        num_sec=3600,
-    )
-    view.switch_to("Completed Plans")
-    view.wait_displayed()
-    migration_plan_collection.find_completed_plan(migration_plan)
+    assert migration_plan.in_progress
+    assert migration_plan.completed
+    assert migration_plan.successful
```

Comment:
these 3 asserts looks a bit odd. does this mean that migration plan is in all 3 states at the same time ?

Reply:
@izapolsk I believe that it is asserting that it goes through the 3 states. For example, looking at `in_progress` there is a wait: https://github.com/ManageIQ/integration_tests/blob/master/cfme/v2v/migration_plans.py#L381

Reply:
No @izapolsk, migration plan will never be in 3 states at same time.

I added 3 `assert` since I want this test to fail right on that moment if any of condition fails, I'm checking if plan is in progress (this will take 15-30min), if that plan is moved to completed tab (second line) and then if completed plan is success (last assert).

Reply:
@jawatts, @Yadnyawalkya, sure thing, I  had the same suspicion but I wanted to lead you to another issue :). I think these properties are incorrectly named. their names should explain what is behind. @Yadnyawalkya, can you rename those in one of next PRs ?